pkg/api/validation/validation_test.go

```diff
@@ -1690,7 +1690,7 @@ func TestValidateService(t *testing.T) {
 			name: "invalid load balancer protocol 2",
 			tweakSvc: func(s *api.Service) {
 				s.Spec.CreateExternalLoadBalancer = true
-				s.Spec.Ports = append(s.Spec.Ports, api.ServicePort{Name: "p", Port: 12345, Protocol: "UDP"})
+				s.Spec.Ports = append(s.Spec.Ports, api.ServicePort{Name: "ilbp2", Port: 12345, Protocol: "UDP"})
```

Comment:
instead of "ilbp2" which seems random (I know it is the test case), can this just be "q" - we use "p" and "q" for a lot of two-port cases.